.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "extends": ["airbnb-base"],
+  "rules": {
+    "no-shadow": "off",
+    "no-param-reassign": "off",
+    "eol-last": "off",
+    "import/extensions": [
+      1,
+      {
+        "js": "always",
+        "json": "always"
+      }
+    ]
+  },
+  "ignorePatterns": ["dist/", "build/"]
+}
```

Comment:
- [ ] It's nice to see that you tried setting up your `.eslint.json` file but it isn't running yet. You should have 5 tests running, not 4. Kindly check again to ensure it runs. It will help you to maintain JavaScript best practices in your code :hugs:
- Check the screenshot:
![image](https://user-images.githubusercontent.com/99715640/223448707-09a14f5f-1413-41e2-8341-95eabe5e02dc.png)

Reply:
Fixed.